app/client/ui/HomeLeftPane.ts

```diff
@@ -146,6 +146,14 @@ export function createHomeLeftPane(leftPanelOpen: Observable<boolean>, home: Hom
           ) : null
         ),
         createHelpTools(home.app),
+       (commonUrls.termsOfService ?
```

Comment:
Indentation seems to be off by one space here:
```suggestion
        (commonUrls.termsOfService ?
```